<launch>
	<arg name="depth_topic"  default="/camera/depth/image_raw"/>
	<arg name="depth_out_image_topic" default="/camera/depth/image"/>
	<arg name="depth_out_dif_topic" default="/camera/depth/image_dif"/>
	<arg name="project_path" default="$(find ros_visual)" />
	<arg name="playback_topics"  default="false"/>
	<arg name="display"  default="true"/>
	<arg name="min_depth"  default="0"/>
	<arg name="max_depth"  default="6000"/>
	<arg name="hor_field"  default="58"/>
	<arg name="ver_field"  default="45"/>
	<arg name="depth_width"  default="640"/>
	<arg name="depth_height"  default="480"/>

	<node pkg="depth" type="depth" name="depth" output="screen" >
		<param name="project_path" value="$(arg project_path)"/>
		<param name="depth_topic" value="$(arg depth_topic)" />
		<param name="depth_out_image_topic" value="$(arg depth_out_image_topic)" />
		<param name="depth_out_dif_topic" value="$(arg depth_out_dif_topic)" />
		<param name="depth_width"  value="$(arg depth_width)"/>
		<param name="depth_height"  value="$(arg depth_height)"/>
		<param name="min_depth"  value="$(arg min_depth)"/>
		<param name="min_depth"  value="$(arg min_depth)"/>
		<param name="hor_field"  value="$(arg hor_field)"/>
		<param name="ver_field"  value="$(arg ver_field)"/>
		<param name="playback_topics" type="bool" value="$(arg playback_topics)"/>
		<param name="display" type="bool" value="$(arg display)"/>
	</node>
</launch>
